Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_Address_Object.yml

```diff
@@ -0,0 +1,493 @@
+  "3":
+    id: "3"
+    taskid: 060c4380-0f99-4a04-8338-c44b6cbdaa4b
+    type: title
+    task:
+      id: 060c4380-0f99-4a04-8338-c44b6cbdaa4b
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1500
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 5f4a9cd4-8f7b-4170-800d-bc9ee601c150
+    type: condition
+    task:
+      id: 5f4a9cd4-8f7b-4170-800d-bc9ee601c150
+      version: -1
+      name: Do the IPs exist?
+      description: Do the IPs exist?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "5"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: notIn
+          left:
+            value:
+              simple: inputs.IP
+            iscontext: true
+          right:
+            value:
+              simple: PrismaSase.Address.address_value
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 690,
+          "y": 600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 9b9c246e-2b37-422b-81c4-cdc836d6b1d2
+    type: regular
+    task:
+      id: 9b9c246e-2b37-422b-81c4-cdc836d6b1d2
+      version: -1
+      name: Create new address object
+      description: Create a new address object.
+      script: '|||prisma-sase-address-object-create'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      address_value:
+        complex:
+          root: inputs.IP
+          filters:
+          - - operator: notIn
+              left:
+                value:
+                  simple: inputs.IP
+                iscontext: true
+              right:
+                value:
+                  simple: PrismaSase.Address.address_value
+                iscontext: true
+          transformers:
+          - operator: uniq
+      name:
+        complex:
+          root: inputs.IP
+          filters:
+          - - operator: notIn
+              left:
+                value:
+                  simple: inputs.IP
+                iscontext: true
+              right:
+                value:
+                  simple: PrismaSase.Address.address_value
+                iscontext: true
+          transformers:
+          - operator: uniq
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+      type:
```

Comment:
You need to verity the type isnt empty before

Reply:
Changed the PB inputs to be mandatory